README.md

```diff
@@ -0,0 +1,25 @@
+# Smaily Drupal module
+
+Smaily email marketing and automation extension plugin for Drupal.
+Includes a simple sign-up form to add new subscribers to Smaily subscribers list.
+## Features
+### Drupal Newsletter Subscription
+- Add new subscribers to Smaily subscribers list.
+- Select autoresponder to send automated emails.
+
+## Requirements
+- Drupal 8+
```

Comment:
Do not add plus sign after Drupal 8, because that would mean that this module works with future Drupal 9 and later.